cryptography/fuzz/fuzz_targets/blake3_hasher.rs

```diff
@@ -88,9 +88,6 @@ fn fuzz_diff_hash(data: &[u8]) {
 }
 
 fn fuzz_digest_operations(data: &[u8]) {
-    let empty_digest = OurBlake3::EMPTY;
```

Comment:
I opted to remove this rather than replace it because I'm not sure that asserting the length of a constant buys us anything